File: backend/web-core/models/book.py
from pydantic import BaseModel

class BookInfo(BaseModel):
    name: str
    category: str | None = None
    author: str | None = None 
    price: int
    publish_year: int | None = None
    publish_company: str | None = None

class Book(BaseModel):
    book_id: int
    quantity: int

File: frontend/src/Models/book.js
export default class Book {
    constructor(name, author, category, publish_year, publish_company, price) {
        this.name = name;
        this.category = category;
        this.author = author;
        this.publish_company = publish_company;
        this.publish_year = publish_year;
        this.price = price;
    }
}